static/js/src/advantage/subscribe/checkout/components/BuyButton/BuyButton.tsx

```diff
@@ -79,6 +80,19 @@ const BuyButton = ({ setError, quantity, product, action }: Props) => {
       return;
     }
 
+    window.plausible("pro-purchase", {
```

Comment:
Should this be on the onsuccess block of the mutation? This way it's possible we report a purchase on plausible that actually failed right?